lib/facebook/setup.js

```diff
@@ -69,13 +69,14 @@ module.exports = function fbSetup(api, bot, logError, optionalParser, optionalRe
             console.log(`\n\n${color.green}Facebook Messenger setup${color.reset}\n`);
             console.log(`\nFollowing info is required for the setup, for more info check the documentation.\n`);
             console.log(`\nYour webhook URL is: ${color.cyan}${lambdaDetails.apiUrl}/facebook${color.reset}\n`);
-            console.log(`Your verify token is: ${color.cyan}${token}${color.reset}\n`);
+            console.log('\nYour verify token is: ${color.cyan}${token}${color.reset}\n`);
```

Comment:
Travis build is failing with 'unterminated string' due to the mismatch between the opening quote and closing back-tick.
```suggestion
            console.log(`\nYour verify token is: ${color.cyan}${token}${color.reset}\n`);
```